ninja-servlet/src/main/java/ninja/servlet/ContextImpl.java

```diff
@@ -332,6 +331,17 @@ public boolean hasCookie(String cookieName) {
 
         return ninjaCookies;
     }
+    
+    @Override
+    public void addCookie(Cookie cookie) {
```

Comment:
We need a test for both addCookie and unsetCookie. Can be mocked. But we need it to make sure the feature does not get removed or crippled by accident :)